tests/system/apiserver-reconnection.sh

```diff
@@ -63,26 +70,26 @@ NS_NAME=chocolate
 create_namespace ${NS_NAME}
 
 # Check for that namespace in etcd.
-docker exec st-etcd etcdctl ls --recursive /calico | grep ${NS_NAME}
+assert_namespace_exists ${NS_NAME}
 
 for n in `seq 0 9`; do
 
     # Stop k8s API
     make stop-k8s-apiserver
 
-    # Wait 60 seconds
-    sleep 60
+    # Wait 30 seconds
```

Comment:
As sad as having a 60 second sleep is, I think we should leave the sleeps alone for this test since I think they're important.

Reply:
done.